app/models/dated_struct.rb

```diff
@@ -13,9 +13,11 @@ def initialize(hash = nil, options = {})
     end
   end
 
+  # :nocov:
   def []=(name, value)
     modifiable?[new_ostruct_member!(name)] = value_or_time(value)
   end
+  # :nocov:
```

Comment:
let's put a # TODO to remind us to remove this method once all reference to it has been removed from other services.